code/implementation/sample-setup/src/main/java/io/cattle/platform/sample/data/SampleDataStartupV5.java

```diff
@@ -0,0 +1,31 @@
+package io.cattle.platform.sample.data;
+
+import static io.cattle.platform.core.model.tables.MachineDriverTable.*;
+
+import io.cattle.platform.core.constants.CommonStatesConstants;
+import io.cattle.platform.core.model.Account;
+import io.cattle.platform.core.model.MachineDriver;
+
+import java.util.List;
+
+public class SampleDataStartupV5 extends AbstractSampleData {
+
+    @Override
+    protected String getName() {
+        return "sampleDataVersion5";
+    }
+
+    @Override
+    protected void populatedData(Account system, List<Object> toCreate) {
```

Comment:
Since they are happening in the same commit, i think we can combine sample data V4 and V5.